### PR TITLE
GeometryStream query and update using flatbuffer schema.

### PR DESCRIPTION
**Pull request by bbastings**

### CODE REVIEW

**Review by pmconne — APPROVED** [review]
How were the contents of ElementGeometryFB.ts generated?

File: core/backend/src/IModelDb.ts [hunk 1]
```diff
@@ -1147,6 +1147,20 @@ export abstract class IModelDb extends IModel {
   public exportPartGraphics(exportProps: ExportPartGraphicsOptions): DbResult {
     return this.nativeDb.exportPartGraphics(exportProps);
   }
+
+  /** Request geometry stream information from an element.
+   * @alpha
+   */
+  public elementGeometryRequest(requestProps: ElementGeometryRequest): DbResult {
```
Comment (pmconne) [inline core/backend/src/IModelDb.ts:1154]: nitpick but these names are awkward. Wouldn't `updateElementGeometry` and `request/processElementGeometry` make more sense?

**Comment by bbastings** [comment]
> How were the contents of ElementGeometryFB.ts generated?

By running flatc.exe with -T option on fbs schema files (exe and scripts to cleanup namespaces, etc. checked in previously to imodel02 for geomlibs support).

**Review by calebmshafer — APPROVED** [review]

**Comment by calebmshafer** [comment]
@bbastings going to override the build requirement since the imodeljs.imodeljs build succeeded.  I'll need to figure out a better workflow for this.

**Pull request merged**